app/presenters/finder_content_item_presenter.rb

```diff
@@ -45,10 +45,11 @@ def details
       document_noun: schema.fetch("document_noun"),
       document_type: metadata.fetch("format"),
       email_signup_enabled: metadata.fetch("signup_enabled", false),
-      format_name: metadata.fetch("format_name"),
+      filter: metadata.fetch("filter", {}),
+      format_name: metadata.fetch("format_name", nil),
       signup_link: metadata.fetch("signup_link", nil),
       show_summaries: metadata.fetch("show_summaries", false),
-      facets: schema.fetch("facets"),
+      facets: schema.fetch("facets", []),
```

Comment:
Ah yeah. I noticed these were missing some decent defaults so I added them in. Forgot to say why.